components/DateTime.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import ReactDateTime from 'react-datetime';
+import DateTimePicker from 'react-datetime-picker/dist/entry.nostyle';
```

Comment:
Rather than importing the code and CSS separately, what happens if we import `react-datetime-picker` directly (without the CSS)?

Reply:
@Betree : It gives the build error; `Global CSS cannot be imported from within node_modules` which is why I suggested this approach. 🤔

Reply:
@Betree : Here is the relevant build for the commit with the direct import; https://github.com/opencollective/opencollective-frontend/runs/1580735310

Reply:
Thanks for sharing that, looks good then